.github/workflows/ci-cd.yml

```diff
@@ -199,12 +198,7 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.pyver }}
-        allow-prereleases: >-
```

Comment:
@bdraco this should be kept long-term so that adding `-dev` things in the future wouldn't require remembering to reimplement it once again.

Reply:
I matched it to yarl since I wasn't sure if `3.13-dev` would pull non `rc`/`stable` 3.13

https://github.com/aio-libs/yarl/blob/eaf76414d44e72b022a7dee3b48bb532934a2f33/.github/workflows/ci-cd.yml#L226

Reply:
It looks like keeping `-dev` will still work https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#specifying-a-python-version

Reply:
I'll revert that change

Reply:
Just realized there are other checks for `-dev` though that we want to be fatal if 3.13 breaks though....so I'm not sure if reverting it makes sense

Reply:
Okay, leave it as is. I usually like using a different syntax `~3.13.0-0` and check for `~` in other places. But I suppose it's not important to figure out *right now*. Something to remember at some point, perhaps.

I also didn't fully realize that this is about the action input. I typically have this in the check for allowing failures which is what confused me and I overlooked the context.